client/src/res/values/joyride.js

```diff
@@ -45,72 +47,75 @@ export const JOYRIDE_STEPS = [
           spotlightClicks: true,
           disableOverlay:true,
           content: 'Click on the "Back" button return to the previous chart, then click "Next".',
+          hideBackButton: true,
         },      
         {// 7
           target: '.leaflet-states-pane .leaflet-interactive:nth-child(4)',
```

Comment:
The isSameNode error (https://github.com/IDREsandbox/hatecrimemap/pull/60#discussion_r680658884) caused my client to crash from memory overload

Reply:
Looking into this now, in the case of the data table, after you close it the joyride can't 'mount' on it for the step_after part of the step with index 5 which causes the issamenode error to pop up. Maybe there could be a way to advance it forward even though the table isn't technically there to be mounted on anymore

Reply:
Changed step with index 5 to target '.homePage.' It is still placed in the exact same spot at the center of the screen but removes the dependence on the chart being open for the target to mount and also simplifies the callback a little bit.

Reply:
The back button also causes this issue. I'm thinking that I could change it to target sidemenu chart and just place the beacon at top left